assets/js/base/components/cart-checkout/totals/shipping/index.tsx

```diff
@@ -60,15 +61,25 @@ const ShippingAddress = ( {
 	setIsShippingCalculatorOpen,
 	shippingAddress,
 }: ShippingAddressProps ): ReactElement | null => {
+	const hasDefaultCustomerAddress = getSetting(
+		'hasDefaultCustomerAddress',
+		false
+	);
+
+	const formattedLocation = formatShippingAddress( shippingAddress );
+	const label =
+		! hasDefaultCustomerAddress && ! formattedLocation
```

Comment:
@mikejolley Do you think we should remove `hasDefaultCustomerAddress` check as well? I think `formatLocation` null check is enough to determine whether we have the address or not to change the label, irrespective of the user's setting in WC. Let me know your thoughts.

Reply:
Yeah that makes sense, I'm assuming that formatted location logic needs the basics (state, country, city) or will null. If thats the case, yes remove it.